website/scripts/fetch-popular-issues.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Copyright 2025 Enactic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ -z "$GH_TOKEN" ]; then
+  echo "Warning: GH_TOKEN environment variable is not set"
+fi
```

Comment:
Could you remove this?
I feel that this is redundant. If `GH_TOKEN` isn't set, `gh` reports an error.

```suggestion
```

Reply:
fix: [13636be](https://github.com/enactic/openarm/pull/222/commits/13636be3127e43ee6d23ea9993ca1b331bff6b79) We can remove it!